### PR TITLE
display career brief for an organisation instead of description

### PR DESCRIPTION
**Pull request by dstodolny**
* replaced organisation description in the jobs section to career brief defined in prismic

**Important** `badger-brain` [PR](https://github.com/redbadger/badger-brain/pull/101) should go first

### CODE REVIEW

File: shared/components/JobsSection/index.js [hunk 1]
```diff
@@ -5,24 +5,24 @@ const defaultImage = '/img/PNG/partner.png';
 
 const isEmpty = array => !Array.isArray(array) || array.length === 0;
 
-const Job = ({ title, location, description, displayDescription, jobURL }) => (
+const Job = ({ title, location, description, displayCareerBrief, jobURL }) => (
   <div className="Job">
     <h4 className="Job__title">{title}</h4>
     <p className="Job__location">{location}</p>
-    {displayDescription && <p className="Job__description">{description}</p>}
+    {displayCareerBrief && <p className="Job__description">{description}</p>}
```
Comment (asavin) [inline shared/components/JobsSection/index.js]: i think this change is not needed

File: shared/components/JobsSection/index.js [hunk 2]
```diff
@@ -5,24 +5,24 @@ const defaultImage = '/img/PNG/partner.png';
...
   </div>
 );
 
 Job.propTypes = {
   title: PropTypes.string,
   location: PropTypes.string,
-  description: PropTypes.string,
-  displayDescription: PropTypes.bool,
+  careerBrief: PropTypes.string,
+  displayCareerBrief: PropTypes.bool,
```
Comment (asavin) [inline shared/components/JobsSection/index.js]: same here

**Comment by gregdorward** [comment]
I'm getting a 500 server error when trying to access the build generated by this PR - does anyone know what might be causing that? I've tried rebuilding in CircleCI, but that had no effect

**Comment by Etiene** [comment]
@gregdorward this is probably because badger brain needs to be deployed first.
cc @asavin

**Comment by gregdorward** [comment]
@stevecottle could you take a look at the output of this one to see if you're happy with it please? The career brief shown in this screenshot replaces the description we are currently displaying

screen shot 2017-02-23 at 14 48 28

**Comment by gregdorward** [comment]
This one still needs a code review, as it currently has no approvers

**Comment by gregdorward** [comment]
Holding off on merging until we have the copy for career briefs set up in production

**Comment by gregdorward** [comment]
{
        "name": "Bloomberg",
        "careerBrief": null
      }

**Comment by dstodolny** [comment]
Oh yeah. We should test it locally using production prismic data. You should make amendments in your `.env` file.

**Comment by gregdorward** [comment]
@dstodolny, you mentioned ASOS, but the following companies also don't have any data returned when running the query against Badger Brain in production. I just wanted to check if this is expected?  

`    {
        "name": "Bloomberg",
        "careerBrief": null
      },
{
        "name": "Magnolia CMS",
        "careerBrief": null
      },
      {
        "name": "DevelopHer",
        "careerBrief": null
      },
      {
        "name": "Bitgenics",
        "careerBrief": null
      },
      {
        "name": "Red Badger",
        "careerBrief": null
      },
      {
        "name": "Ticketmaster",
        "careerBrief": null
      },
      {
        "name": "Twilio",
        "careerBrief": null
      },
      {
        "name": "Zalando",
        "careerBrief": null
      },
      {
        "name": "Code Your Future",
        "careerBrief": null
      }, `

**Comment by gregdorward** [comment]
Ok, I've tested this one running the branch locally against production data from Badger Brain. The careerBrief data is being displayed where the description was once held - This one is good to be merged

**Pull request merged**